client/src/Components/Star/StarRating.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { FaStar } from 'react-icons/fa';
+import { Input } from './StarStyles';
+
+const StarRating = () => {
+  const [rating, setRating] = useState(0);
```

Comment:
Wydaje mi się że to będziemy pobierali ze stora, ale na razie może tak chyba zostać, później będziemy zmieniać :D